governance/package.json

```diff
@@ -21,8 +23,9 @@
     "@unlock-protocol/hardhat-helpers": "workspace:^",
     "@unlock-protocol/hardhat-plugin": "workspace:^",
     "@unlock-protocol/networks": "workspace:./packages/networks",
+    "arb-shared-dependencies": "1.0.0",
     "eslint": "8.54.0",
-    "ethers": "6.10.0",
+    "ethers": "6.11.1",
```

Comment:
Is that upgrade needed for the PR to work ? 

If not could you remove the change in version (better to have it in a separate PR as it may break other scripts on the repo).